packages/node-resolve/test/browser.js

```diff
@@ -190,7 +190,7 @@ test('supports `false` in browser field', async (t) => {
   await testBundle(t, bundle);
 });
 
-test('pkg.browser with mapping to prevent bundle by specifying a value of false', async (t) => {
+test.only('pkg.browser with mapping to prevent bundle by specifying a value of false', async (t) => {
```

Comment:
```suggestion
test('pkg.browser with mapping to prevent bundle by specifying a value of false', async (t) => {
```

Apparently this change has not been comitted to the branch...